src/omv/sensors/genx320.c

```diff
@@ -84,6 +84,8 @@ static bool hot_pixels_disabled = false;
 static int32_t contrast = CONTRAST_DEFAULT;
 static int32_t brightness = BRIGHTNESS_DEFAULT;
 
+AFK_HandleTypeDef psee_afk;
```

Comment:
```suggestion
static AFK_HandleTypeDef psee_afk;
```
Is this state really needed? It seems transient and can probably be on the stack instead of reserving memory.

Reply:
> ```suggestion
> 
> static AFK_HandleTypeDef psee_afk;
> 
> ```
> 
> Is this state really needed? It seems transient and can probably be on the stack instead of reserving memory. Maybe it is to check if it's active. In this case, it should be static.

Yes, I used it to check the state and not re-initialize it several times.
Sorry I cannot test it this week (on vacation without a laptop)